website/pages/en/managing/transferring-subgraph-ownership.mdx

```diff
@@ -24,16 +27,35 @@ Or a wallet explorer like **Rainbow.me**:
 https://rainbow.me/your-wallet-addres
 ```
 
-## Transferring ownership of a subgraph
+### Step-by-Step
 
-To transfer ownership of a subgraph, you can use the UI built into Subgraph Studio:
+To transfer ownership of a subgraph, do the following:
 
-![Subgraph Ownership Transfer](/img/subgraph-ownership-transfer-1.png)
+1.  Use the UI built into Subgraph Studio: ![Subgraph Ownership Transfer](/img/subgraph-ownership-transfer-1.png)
 
-And then choose the address that you would like to transfer the subgraph to:
+2.  Choose the address that you would like to transfer the subgraph to: ![Subgraph Ownership Trasfer](/img/subgraph-ownership-transfer-2.png)
```

Comment:
To prevent the items' text from being stuck to the images:

```mdx
1.  Use the UI built into Subgraph Studio:

    ![Subgraph Ownership Transfer](/img/subgraph-ownership-transfer-1.png)

2.  Choose the address that you would like to transfer the subgraph to:

    ![Subgraph Ownership Trasfer](/img/subgraph-ownership-transfer-2.png)
```

### Before

![CleanShot 2024-08-14 at 11 06 05@2x](https://github.com/user-attachments/assets/2960b888-610a-43e8-aef9-3b5cb3a523c6)


### After

![CleanShot 2024-08-14 at 11 05 40@2x](https://github.com/user-attachments/assets/01c3bec7-d420-4b6c-8a1b-588f97d4f311)